<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlownUpParamDisplay</class>
 <widget class="QDialog" name="BlownUpParamDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Blown-up Display</string>
  </property>
  <widget class="QSliderMod1" name="verticalSlider_blownup">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>31</width>
     <height>571</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEditMod1" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>311</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>70</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblParam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Param</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblBlock">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>451</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Block</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="qsynthKnob" name="dial_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>523</y>
     <width>249</width>
     <height>178</height>
    </rect>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="pageStep">
    <number>32</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_MaxVal">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_MinVal">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>660</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblController">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>301</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>40</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string>Controller</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSliderMod1</class>
   <extends>QSlider</extends>
   <header>qslidermod1.h</header>
   <slots>
    <slot>setValueFromText(QString)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>QLineEditMod1</class>
   <extends>QLineEdit</extends>
   <header>qlineeditmod1.h</header>
   <slots>
    <slot>setTextFromInt(int)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qsynthKnob</class>
   <extends>QDial</extends>
   <header>qsynthKnob.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
